code/modules/modular_computers/file_system/program.dm

```diff
@@ -182,4 +182,4 @@
 	. = ..()
 	if(.)
 		return
-	return TRUE
+	return .
```

Comment:
should we add `return .` to 184 as well if we're going to be explicit with the value here? actually, this just seems fully redundant now

Reply:
I just changed it because TRUE meant that all subtypes would early return and UIs wouldnt work, doesn't really matter.

Reply:
yeah but it just seems like the whole ui_act thing here is completely redundant since all it does is call parent and return the value parent returns with practically zero uniqueness

Reply:
's gone